v2/schema/example.jsonld

```diff
@@ -0,0 +1,89 @@
+{
+  "@context": "https://raw.githubusercontent.com/geoblacklight/geoblacklight-schema/json-ld-schema/v2/schema/context.jsonld",
+  "@id": "http://hdl.handle.net/2451/34502",
+  "@type": "dcat:Dataset",
+  "accessLevel": "public",
+  "creator": [
+    "GIS Lab, Newman Library, Baruch CUNY"
+  ],
+  "description": "The subway complexes layer was created to represent ridership data for the NYC subway system (Metropolitan Transportation Authority, or MTA).",
+  "distribution": [
+    {
+      "@type": "dcat:Distribution",
+      "conformsTo": "http://www.opengis.net/def/serviceType/ogc/wms",
+      "accessURL": "https://maps-public.geo.nyu.edu/geoserver/sdr/wms",
+      "layerId": "sdr:nyu_2451_34502",
+      "describedBy": "https://archive.nyu.edu/retrieve/74755/nyu_2451_34502_doc.zip",
+      "describedByType": "application/pdf",
+      "description": "2015 New York City Subway Complexes and Ridership WMS Service"
+    },
+    {
+      "@type": "dcat:Distribution",
+      "conformsTo": "http://www.opengis.net/def/serviceType/ogc/wfs",
+      "accessURL": "https://maps-public.geo.nyu.edu/geoserver/sdr/wfs",
+      "layerId": "sdr:nyu_2451_34502",
+      "describedBy": "https://archive.nyu.edu/retrieve/74755/nyu_2451_34502_doc.zip",
+      "describedByType": "application/pdf",
+      "description": "2015 New York City Subway Complexes and Ridership WFS Service"
+    },
+    {
+      "@type": "dcat:Distribution",
+      "conformsTo": "http://www.esri.com/library/whitepapers/pdfs/shapefile.pdf",
+      "downloadURL": "https://archive.nyu.edu/retrieve/74701/nyu_2451_34502.zip",
+      "describedBy": "https://archive.nyu.edu/retrieve/74755/nyu_2451_34502_doc.zip",
+      "describedByType": "application/pdf",
+      "format": "ESRI Shapefile",
+      "mediaType": "application/zip; ogr-format=\"ESRI Shapefile\"",
+      "description": "2015 New York City Subway Complexes and Ridership Shapefile",
+      "title": "Original Shapefile"
+    },
+    {
+      "@type": "dcat:Distribution",
+      "conformsTo": "http://www.isotc211.org/schemas/2005/gmd/",
+      "downloadURL": "http://metadata.geo.nyu.edu/records/edu.nyu/handle/2451/34502/iso19139.xml",
+      "format": "ISO19139",
+      "mediaType": "application/xml; schema=iso19139",
+      "description": "ISO19139 metadata for 2015 New York City Subway Complexes and Ridership"
+    }
```

Comment:
there's often an HTML version of the metadata that we could include as an example.